dsc/src/util.rs

```diff
@@ -406,16 +407,30 @@ pub fn get_input(input: &Option<String>, stdin: &Option<String>, path: &Option<S
     parse_input_to_json(&value)
 }
 
-pub fn set_dscconfigroot(config_path: &str)
+pub fn absolutize_and_set_dscconfigroot(config_path: &str) -> String
```

Comment:
I think this function should just be called `set_dscconfigroot` and making it absolute is just a comment than part of the name

Reply:
Updated.